Python/README.md

```diff
@@ -1,3 +1,91 @@
+    <img alt="Github All Contributors" src="https://img.shields.io/github/all-contributors/tesseractcoding/neoalgo">
+</p>
+
+# Codes in the Python language
+
+## Data Structures
+* [Queue](ds/Queue.py)
+* [Stacks](ds/Stacks.py)
+* [Tree (includes Traversals - in,pre,post-order)](ds/Tree_Traversals_In_Pre_Post_Order.py)
+* [Tree (Level Order Traversal)](ds/Tree_Level_Order_Traversal.py)
+* [Reverse Linked List](ds/Reverse_linked_list.py)
+
+## Searching
+* [Bilinear Search](search/bilinear.py)
+* [Binary Search](search/binarysearch.py)
+* [Breadth First Search](search/bfs.py)
+* [Depth First Search](search/dfs.py)
+* [Linear Search](search/Linear_Search.py)
+* [Random Search Las vegas version](search/random_search_las_vegas.py)
+* [Sublinear Search](search/Sublinear_search.ipynb)
+* [Exponential Search](search/exponential_search.py)
+
+## Sorting
+* [Bead Sort](sort/Bead_Sort.py)
+* [Bubble Sort](sort/Bubble_Sort.py)
+* [Heap Sort](sort/Heap_sort.py)
+* [Insertion Sort](sort/insertion.py)
+* [Merge Sort](sort/Merge_Sort.py)
+* [Quick Sort](sort/QuickSort.py)
+* [Radix Sort](sort/Radix_Sort.py)
+* [Selection Sort](sort/Selection_Sort.py)
+* [Topological Sort](sort/topological_sort.py)
+* [Intro Sort](sort/IntroSort.py)
+* [Merge Sort Sequantial](sort/Sequential_MergeSort.py)
+* [Merge Sort Parallel](sort/Parallel_MergeSort.py)
+
+## Graphs
+* [Depth of Tree](graphs/Depth_Of_Tree.py)
+* [Height of Tree](graphs/height_of_given_tree.py)
+* [Prim's Algorithm](graphs/Prim_Algorithm.py)
+* [Kruskal's Algorithm](graphs/Kruskal_Algorithm.py) 
+* [Directed Acyclic Graph](graphs/Directed_Acyclic_Graph.py)
+
+## Machine Learning
+* [Decision Tree](ml/DecisionTree_From_Scratch.ipynb)
+* [Iris Classifier using Keras](ml/Iris_Classifier_using_Keras.ipynb)
+* [K Nearest Neighbor](ml/K_nearest_neighbors.ipynb)
+* [Linear Regression](ml/Linear_Regression.ipynb)
+* [Linear Regression using PyTorch](ml/Linear_Regression_using_PyTorch.ipynb)
+* [Logistic Regression](ml/Logistic_Regression.ipynb)
+* [Numeric Outlier Selection](ml/Numerical_outlier_method.ipynb)
+* [Principal Component Analysis](ml/principal_component_analysis.ipynb)
+
+## Mathematics
+* [Area](math/Area.py)
+* [Binary Exponentiation](math/Binary_Exponentiation.py)
+* [GCD - Euclidean Algorithm](math/GCD_Euclidean_Algorithm.py)
+* [Factorial](math/factorial.py)
+* [Fibonacci Sequence Recursion](math/fibonacci_recursion.py)
+* [Sieve of Eratosthenes](math/prime_number_using_sieve_of_Eratosthenes.py)
+* [Fibonacci number](math/Fibonacci.py)
+* [Strassen's Matrix Multiplication](math/Strassen's_Matrix_Mul.py)
+
+## Competitive Programming
+
+* [Rabin Karp Algorithm](cp/Rabin_Karp.py)
+
+## Dynamic Programming
+_add list here_
+
+## Blockchain
+_add list here_
+
+
+## Cryptography
+_add list here_
+
+## Other
+* [Check Inversion](other/Check_Inversion.py)
+* [Kadane's Algorithm](other/Kadanes_Algorithm.py)
+* [Reverse a string using stack](other/Reverse_string_using_stack.py)
+=======
```

Comment:
Why have you made so many changes in README file?
Revert all the changes and just add links of your contributions.